### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by Stanleeeeee**
In this project, l created a form of UI for my school library app. This way it can be invoked as an executable and not something you use in IRB exclusively.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -7,19 +8,20 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```
Comment (mikethreels) [inline person.rb:11]: - [x] due to setting up the default parent permission like this `parent_permission: true` the user will receive the below error when creating a person please replace it by `parent_permission = true`

obraz

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+  def create_teacher
+    print 'Specialization: '
+    specialization = gets.chomp
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]'
+    permission = gets.chomp.upcase
```
Comment (mikethreels) [inline app.rb]: - [ ] for the teacher it is not necessary to check parent permission.

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    @rentals.push(rental)
+    puts 'Rental created successfully 👋📦'
+    run
+  end
+
+  def list_rental
+    print 'ID of person: '
+    id = gets.chomp.to_i
+    rentals = @rentals.select { |rental| rental.person.id == id }
+    p rentals
```
Comment (mikethreels) [inline app.rb]: - [ ] due to this line of code the output when listing the rentals is not correct, please remove it.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Stanley, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (Stanleeeeee) [reply app.rb]: @mikethreels Thank you for your review, Do you want me to remove the permission from parents?

File: person.rb (hunk 1 above)
Comment (Stanleeeeee) [reply person.rb:11]: @mikethreels If I use parent_permission = true, i get errors in rubocop linters saying its boolean that I use  :true instead
TSE
.

File: app.rb (hunk 2 above)
Comment (mikethreels) [reply app.rb]: hi @Stanleeeeee, 
Yes but only when creating a teacher the students still need to get parent permission

File: person.rb (hunk 1 above)
Comment (mikethreels) [reply person.rb:11]: Hi @Stanleeeeee,
I wasn't aware that rubocop gave this error, in that case you will need to change it back, however make sure the bug is also fixed.

**Review by mugishasam123 — APPROVED** [review]
Hi @Stanleeeeee  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive PR title 👍 
- Nice use of github flow 🥇 
- No linter errors , keep rocking 🚀 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**